### ISSUE TITLE
[NAPI] extensions install problem

### ISSUE THREAD

**Issue by juntao** [WasmEdge/WasmEdge#506]
If I explicitly specify to install 0.8.2, it still tries (and fails) to install 0.9.0 rc1

```
$ npm install wasmedge-extensions@0.8.2
... ...
Fetching WasmEdge-tensorflow-0.9.0~rc.1

    Please see --help
    If issue perists make a trace using -V and submit it to
https://github.com/WasmEdge/WasmEdge/issues/new?assignees=&labels=&template=bug_report.md
```

**Comment by tpmccallum** [comment]
Also found a similar issue where the installer just stops due to $VERSION variable not being in scope.

A fix has been created at https://github.com/WasmEdge/WasmEdge/pull/511/files

**Comment by SAtacker** [comment]
This should have been fixed in a13d8ba465a11f64a70a92ce94657238e6789320 because of https://github.com/WasmEdge/WasmEdge/commit/a13d8ba465a11f64a70a92ce94657238e6789320#diff-001b8193af60259ffc2829010784899f35764c864992fe3e96a606ad2650bb2bL93
